js/lowebutil.js

```diff
@@ -15,7 +15,7 @@ function getParameterByName(name, url) { // eslint-disable-line no-unused-vars
 }
 
 function isElectron() {
-  return window && window.process && window.process.type;
+  return window?.process?.type;
```

Comment:
Optional chaining is a good feature for clear code. But i'm afaid that is still new for widest browser support.

According to https://caniuse.com/mdn-javascript_operators_optional_chaining , only chrome/firefox version since ealier 2020 will support. 

So I think by now we still need to support users from old browser.

Reply:
The reason why I switched to optional chaining is because I saw the use of media session api. Which is nearly as new as optional chaining. Chrome only supports media session api in 80. Beta in chrome://flag in 73. So I believe getting that feature will not making such a compatible issue.

Reply:
ok, i got it. The key reason is failover ability. If media session api is not available, extension can still compile and run, but I'm not sure if optional chaining will work if chrome not support it.

Reply:
You are absolutely correct ! optional chaining will immediately throw if it's not supported. I will revert the change on optional chaining. Also, could you please take a look more on #444. The media control is not working quite well with the play bar.

Reply:
Sure. I'll take a loot at it and let's discuss there.